internal/v2/cert_verifier/cert_verifier_test.go

```diff
@@ -65,6 +67,16 @@ func TestVerifyClientCertChain(t *testing.T) {
 		log.Fatalf("error starting fake S2Av2 Server: %v", err)
 	}
 
+	os.Setenv("S2A_ACCESS_TOKEN", "token")
+	accessTokenManager, err := tokenmanager.NewSingleTokenAccessTokenManager()
+	if err != nil {
+		log.Fatalf("error generating token to authenticate to s2a: %v", err)
```

Comment:
nit: I think this should be "t.Fatalf" here, and similarly throughout the tests. Sorry if I missed this in previous reviews.

Reply:
Will do! Changed this from log to t in all three test suites